samples/word/01-basics/insert-formatted-text.yaml

```diff
@@ -6,39 +6,46 @@ host: WORD
-                    // Here we go to insert first sentence. Then we adjust the formatting.
-                    // Note that replace affects the calling object, in this case the entire document body.
-                    // A similar method can also be used at the range level.
-                    var firstSentence = context.document.body.insertText("This is some formatted text!", "replace");
-                    firstSentence.font.name = "Courier New";
-                    firstSentence.font.bold = true;
-                    firstSentence.font.size = 18;
+            await Word.run(async (context) => {
+                // Insert first sentence. Then we adjust the formatting.
+                // Note that replace affects the calling object, in this case the entire document body.
+                // A similar method can also be used at the range level.
+                let firstSentence = context.document.body.insertText("This is some formatted text!", "replace");
+                firstSentence.font.name = "Courier New";
+                firstSentence.font.bold = true;
+                firstSentence.font.size = 18;
+
+                // Second sentence, let's insert it as a paragraph after the previously inserted one.
+                let secondSentence = context.document.body.insertParagraph("This is other formatted text.", "end");
+                secondSentence.font.bold = false;
+                secondSentence.font.italic = true;
+                secondSentence.font.name = "Berlin Sans FB";
+                secondSentence.font.size = 30;
+                secondSentence.font.color = "blue";
 
-                    // Second sentence, let's insert it after the previously inserted one.
-                    var secondSentence = context.document.body.insertText("This is other formatted text.", "end");
-                    secondSentence.font.bold = false;
-                    secondSentence.font.italic = true;
-                    secondSentence.font.name = "Berlin Sans FB";
-                    secondSentence.font.size = 30;
-                    secondSentence.font.color = "blue";
+                // You can also use pre-exsiting styles.
+                let lastSentence = context.document.body.insertParagraph("To be or not to be", "end");
 
-                    // You can also use pre-exsiting styles.
-                    var lastSentence = context.document.body.insertParagraph("To be or not to be", "end");
+                // Use styleBuiltIn to use an enumeration of existing styles. If your style is custom make sure to use range.style = "name of your style";
+                lastSentence.styleBuiltIn = Word.Style.intenseReference;
 
-                    // Use styleBuiltIn to use an enumeration of existing styles. If your style is custom make sure to use range.style = "name of your style";
-                    lastSentence.styleBuiltIn = Word.Style.intenseReference;
+                await context.sync();
+                console.log("success");
```

Comment:
Since users will see the sentence added, is this console.log("success") really needed?

Reply:
local fixed